scripts/fonts/chikarego2-polish.py

```
@@ -0,0 +1,87 @@
+    f.appendSFNTName("English (US)", key, NAME)
+# CC-BY: attribution must be preserved
+f.copyright = ("%s: ChiKareGo2 by Giles Booth (CC BY), "
+               "extended with Polish diacritics." % NAME)
+f.appendSFNTName("English (US)", "License",
+                 "Creative Commons Attribution (CC BY). Original: ChiKareGo2 by Giles Booth.")
+f.appendSFNTName("English (US)", "License URL",
+                 "https://creativecommons.org/licenses/by/4.0/")
+
+for ext in ("woff", "woff2"):
+    f.generate("public/fonts/CziKareChodz3.%s" % ext)
+os.makedirs("/tmp/fontpatch/out", exist_ok=True)
```

Ruff (0.15.17)

[error] Probable insecure usage of temporary file or directory: "/tmp/fontpatch/out"

(S108)